### PR TITLE
clearning existing controls when handleWCONPROD

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

File: src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -757,6 +757,9 @@ namespace Opm {
                     auto& well = this->m_wells.at(well_name);
                     bool switching_from_injector = !well.isProducer(currentStep);
                     WellProductionProperties properties(well.getProductionProperties(currentStep));
```
Comment (GitPaean) [inline src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: Only very little information can be transferred through report steps, so making a copy is of very limited usage comparing with creating a new one.

**Comment by GitPaean** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: You are probably right - but that is unfortunate; as the 'carry information forward' idiom is quite central to the Schedule implementation.

**Comment by joakim-hove** [comment]
This even seems to fix the restart issue - going right in ;-) Thank you!

**Pull request merged**